android/app/src/main/java/im/status/ethereum/MainActivity.kt

```diff
@@ -127,6 +130,13 @@ class MainActivity : NavigationActivity(), ActivityCompat.OnRequestPermissionsRe
 
         setSecureFlag()
 
+        val w: Window = window
+        w.statusBarColor = Color.TRANSPARENT
+        w.navigationBarColor = Color.TRANSPARENT
+        w.decorView.systemUiVisibility =
+            View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION or View.SYSTEM_UI_FLAG_LAYOUT_STABLE
```

Comment:
For full screen on Android

Reply:
Very cool that you found this configuration 🙌 
One small note, it might be worth asking in design-discussion if they are aware of any reasons that transparency would not be okay. I can't think of any atm, but it still could be useful to let them know (if you haven't already).

Reply:
They said actually this was the expected way to work:
https://discord.com/channels/1210237582470807632/1274068685266489434/1358865214799610126